_inc/client/plans/single-product-backup.jsx

```diff
@@ -36,3 +81,85 @@ export function PlanRadioButton( {
 		</label>
 	);
 }
+
+// eslint-disable-next-line no-unused-vars
+class SingleProductBackupBody extends React.Component {
+	static propTypes = {
+		backupPlanPrices: PropTypes.object,
+		currencySymbol: PropTypes.string,
+		setSelectedBackupType: PropTypes.func,
+		selectedBackupType: PropTypes.string,
+		upgradeLinks: PropTypes.object,
+	};
+
+	handleSelectedBackupTypeChange = event => {
+		this.props.setSelectedBackupType( event.target.value );
+	};
+
+	render() {
+		const { currencySymbol, backupPlanPrices, selectedBackupType, upgradeLinks } = this.props;
+
+		const upgradeTitles = {
+			'real-time': __( 'Upgrade to Real-Time Backups' ),
+			daily: __( 'Upgrade to Daily Backups' ),
+		};
+
+		const backupOptions = [
+			{
+				type: 'daily',
+				name: __( 'Daily Backups' ),
+				price: backupPlanPrices.jetpack_backup_daily.yearly,
```

Comment:
It feels odd that we're passing an entire `backupPlanPrices ` object to this component and only using pieces of it like this. Are we passing more than we need here?